src/main/java/com/microsoft/sqlserver/jdbc/SQLServerLexer.java

```diff
@@ -20,7 +20,7 @@
 @SuppressWarnings({"all", "warnings", "unchecked", "unused", "cast"})
 class SQLServerLexer extends Lexer {
     static {
-        RuntimeMetaData.checkVersion("4.7.2", RuntimeMetaData.VERSION);
+        RuntimeMetaData.checkVersion("4.9.2", RuntimeMetaData.VERSION);

```

Comment:
what's this for? this doesn't have anything to do with transient errors?

Reply:
That's to fix the build warning we've been getting since bumping the ANTLR dependency from 4.7.2 to 4.9.2.